index.html

```diff
@@ -1,248 +1,292 @@
-            <img src="assets/languages-icon.svg" alt="">
-            <h4>languages</h4>
-
-            <ul class="portfolio-languages">
-                <li>Javascript</li>
-                <li>Ruby</li>
-                <li>HTML</li>
-                <li>CSS</li>
-            </ul>
-        </div>
+      <div class="portfolio-details">
+        <img src="assets/languages-icon.svg" alt="" />
+        <h4>languages</h4>
 
-        <div class="portfolio-details">
-            <img src="assets/icon-frameworks.svg" alt="">
-            <h4>Frameworks</h4>
+        <ul class="portfolio-languages">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
 
-            <ul class="portfolio-languages">
-                <li>React.js</li>
-                <li>Ruby on Rails</li>
-                <li>RSpec</li>
-                <li>Capybara</li>
-                <li>Selenium</li>
-            </ul>
-        </div>
+      <div class="portfolio-details">
+        <img src="assets/icon-frameworks.svg" alt="" />
+        <h4>Frameworks</h4>
 
-        <div class="portfolio-details">
-            <img src="assets/skills-icon.svg" alt="">
-            <h4>Skills</h4>
+        <ul class="portfolio-languages">
+          <li>React.js</li>
+          <li>Ruby on Rails</li>
+          <li>RSpec</li>
+          <li>Capybara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
 
-            <ul class="portfolio-languages">
-                <li>Database Management</li>
-                <li>Version Control</li>
-                <li>Cli</li>
-                <li>Web Development</li>
-                <li>Api Design</li>
-            </ul>
-        </div>
+      <div class="portfolio-details">
+        <img src="assets/skills-icon.svg" alt="" />
+        <h4>Skills</h4>
 
-        
+        <ul class="portfolio-languages">
+          <li>Database Management</li>
+          <li>Version Control</li>
+          <li>Cli</li>
+          <li>Web Development</li>
+          <li>Api Design</li>
+        </ul>
+      </div>
     </section>
 
+    <section id="contact-form">
+        <p>I'm always interested in hearing about new projects, so if you you'd like to chat please get in touch</p>
+        <div class="inputs">
+            <form class="contact" action="https://formspree.io/f/mgebgadd" method="post">
+                <input class="form1" type="text" id="name" type="text" placeholder="Full name" maxlength="30"   required ><br>
+                <input class="form2" name="Email" id="email" type="email" placeholder="Email address"   required><br>
+                <textarea class="text_form" name="textarea" id="textarea" placeholder="Hey I'd like to connect to..." maxlength="500" rows="5"  required></textarea><br>
+                <button class="greenbtn" type="submit" class="greenbtn">Get in touch</button>
+                <span>abdussalamabro@gmail.com</span>
```

Comment:
- [ ] Please 🙏🏻 follow the `HTML` best practices in your project, as you can see that you used the `span` element inside the **form** tag, because your `span` element which holds your email address is not the part of the form, that's why kindly put your `email` contact information outside the *form* tag.

Reply:
> * [ ]  Please 🙏🏻 follow the `HTML` best practices in your project, as you can see that you used the `span` element inside the **form** tag, because your `span` element which holds your email address is not the part of the form, that's why kindly put your `email` contact information outside the _form_ tag.

- [ ] Just as the previous reviewer said here, kindly note that the email link isn't part of the form and should be placed outside it, but also in within an anchor tag will be better. 😃

Reply:
Thanks for highlighting this i updated that and put span tag outside from form